packages/remark-mdx/test/test.js

```diff
@@ -0,0 +1,39 @@
+const unified = require('unified')
+const remarkParse = require('remark-parse')
+const remarkMDX = require('..')
+const mdxAstToMdxHast = require('../../mdx/mdx-ast-to-mdx-hast')
```

Comment:
L3 uses MDX, L4 uses Mdx. Should this be more consistent? no strong feeling but I like L4.